tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-cuda.yml

```diff
@@ -28,8 +29,22 @@ stages:
     torch_version: '2.0.0'
     opset_version: '15'
     cuda_version: '11.8'
+    cuda_version_nodot: '118'
     cmake_cuda_architectures: 70;75;80;86
     docker_file: Dockerfile.manylinux2_28_training_cuda11_8
     agent_pool: Onnxruntime-Linux-GPU
     upload_wheel: 'no'
     debug_build: true
+
+- template: templates/py-packaging-training-cuda-stage.yml
+  parameters:
+    build_py_parameters: --enable_training --update --build
+    torch_version: '2.1.0'
+    opset_version: '15'
+    cuda_version: '12.1'
+    cuda_version_nodot: '121'
+    cmake_cuda_architectures: 60;61;70;75;80;86;90
```

Comment:
cmake_cuda_architectures should be the same as the above.

Reply:
Maybe `70;75;80;86;90` if we want to support H100.